src/main/java/seedu/address/logic/parser/EntryBookParser.java

```diff
@@ -6,15 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.address.logic.commands.ArchivesCommand;
-import seedu.address.logic.commands.BingWebSearchCommand;
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FeedCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.HistoryCommand;
-import seedu.address.logic.commands.ListCommand;
-import seedu.address.logic.commands.SelectCommand;
+import seedu.address.logic.commands.*;
```

Comment:
Don't use star imports

Reply:
Oops, must be IntelliJ who collapsed the imports

Reply:
Fixed in 4b49c71